templates/docs/base/forms.md

```diff
@@ -192,7 +192,7 @@ View example of form validation patterns
 
 ## Required
 
-By applying the class `.is-required` the attribute specifies that an input field must be filled out before submitting the form.
+By applying the class `.is-required` the attribute specifies that an input field must be filled out before submitting the form. You should also add a legend to the form to be explicit about the meaning of the asterisk.
```

Comment:
I've read that active voice is preferred when writing documentation so how about 
```suggestion
Apply the class `.is-required` to mandatory input fields. You should also add a legend to the form to be explicit about the meaning of the asterisk.
```

Reply:
Done 👍🏼